flow/workflows/cdc_flow.go

```diff
@@ -492,10 +490,8 @@ func CDCFlowWorkflow(
 	}
 
 	syncFlowID := GetChildWorkflowID("sync-flow", cfg.FlowJobName, originalRunID)
-	normalizeFlowID := GetChildWorkflowID("normalize-flow", cfg.FlowJobName, originalRunID)
 
 	var restart, finished bool
-	syncCount := 0
```

Comment:
cdc_flow no longer gets notified when sync flows end, need to somehow pass this down to sync flow